swagger.yaml

```diff
@@ -2500,6 +2500,107 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /clusters/{cluster_id}/monitored_operators:
+    get:
+      tags:
+        - operators
+        - installer
+      security:
+        - agentAuth: []
+      description: Lists operators to be monitored for a cluster.
+      operationId: ListOfClusterOperators
+      parameters:
+        - in: path
+          name: cluster_id
+          description: The cluster to return operators for.
+          type: string
+          format: uuid
+          required: true
+        - in: query
+          name: operator_name
+          description: An operator in the specified cluster to return its data.
+          type: string
+          required: false
+      responses:
+        "200":
+          description: Success.
+          schema:
+            $ref: '#/definitions/monitored-operators-list'
+        "401":
+          description: Unauthorized.
+          schema:
+            $ref: '#/definitions/infra_error'
+        "403":
+          description: Forbidden.
+          schema:
+            $ref: '#/definitions/infra_error'
+        "404":
+          description: Error.
+          schema:
+            $ref: '#/definitions/error'
+        "405":
+          description: Method Not Allowed.
+          schema:
+            $ref: '#/definitions/error'
+        "500":
+          description: Error.
+          schema:
+            $ref: '#/definitions/error'
+    put:
+      tags:
+        - operators
+        - installer
+      security:
+        - agentAuth: []
+      description: Controller API to report of monitored operators.
+      operationId: ReportMonitoredOperatorStatus
+      parameters:
+        - in: path
+          name: cluster_id
+          description: The cluster whose operators are being monitored.
+          type: string
+          format: uuid
+          required: true
+        - in: body
+          name: report-params
+          description: The operators monitor report.
+          required: true
+          schema:
+            $ref: '#/definitions/operator-monitor-report'
+      responses:
+        "200":
+          description: Success.
+          schema:
+            $ref: '#/definitions/cluster'
```

Comment:
What's the reason to return the cluster here?

Reply:
You are right, it can be removed.